profile/src/main/java/rdx/works/profile/data/repository/PublicKeyProvider.kt

```diff
@@ -34,24 +35,35 @@ class PublicKeyProvider @Inject constructor(
         entityKind: EntityKind = EntityKind.ACCOUNT
```

Comment:
we should probably pass the `keyKind` here too and use it below, instead of hard coding `Cap26KeyKind.TRANSACTION_SIGNING`, which is wrong for the DEBUG only paths which creates a true ROLA key

Reply:
I don't disagree with that, but ROLA keys are created with `GenerateAuthSigningFactorInstanceUseCase` and don't use the relatively new `PublicKeyProvider`. I will change that to expose the keyKind which will default to `TRANSACTION_SIGNING` which will not be changed by the time being.